kairon/shared/data/data_objects.py

```diff
@@ -895,6 +917,7 @@ class BotSettings(Auditlog):
     data_importer_limit_per_day = IntField(default=5)
     multilingual_limit_per_day = IntField(default=2)
     data_generation_limit_per_day = IntField(default=3)
+    collection_limit = IntField(default=5)
```

Comment:
cognition_collections_limit = 3
cognition_columns_per_collection_limit = 5
we must have both